tests/kafka/kafka.test.ts

```diff
@@ -20,6 +20,9 @@ const kafkaConfig: KafkaConfig = {
   logLevel: logLevel.NOTHING, // FOR TESTING
 }
 const kafka = new KafkaJS(kafkaConfig)
+const patternTopic = /dbos-test-.*/;
```

Comment:
Maybe make this in as a `RegExp` object?

Reply:
It is. Try doting it. I can make it explicit if you wish.

Reply:
How is it that you prefer it ?  new RegExp('dbos-test-.*') or new RegExp(/dbos-test-.*/)?

Reply:
Yeah, making it explicit with the latter notation is good